Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1101,6 +1101,18 @@ public extension Purchases {
 
 #endif
 
+    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+    func handleObserverModeTransaction(
+        purchaseResult: StoreKit.Product.PurchaseResult
+    ) async throws -> StoreTransaction? {
+        guard self.systemInfo.observerMode else {
+            throw NewErrorUtils.observerModeNotEnabledError()
+        }
+        let (_, transaction) = try await self.purchasesOrchestrator.storeKit2TransactionListener.handle(
+            purchaseResult: purchaseResult, fromTransactionUpdate: true)
```

Comment:
I wish we linted this stuff 😇 
I'd format it with symmetrical parenthesis:
```suggestion
        let (_, transaction) = try await self.purchasesOrchestrator.storeKit2TransactionListener.handle(
            purchaseResult: purchaseResult, fromTransactionUpdate: true
        )
```